src/js/send-web-vitals.js

```diff
@@ -2,26 +2,50 @@ function sendWebVitals() {
 
   function sendWebVitalsGAEvents({name, delta, id, attribution, navigationType}) {
 
-    let webVitalInfo = '(not set)';
+    let overrides = {};
 
     switch (name) {
       case 'CLS':
-        webVitalInfo = attribution.largestShiftTarget;
+        overrides = {
+          debug_time: attribution.largestShiftTime,
+          debug_load_state: attribution.loadState,
+          debug_target: attribution.largestShiftTarget || '(not set)',
+        };
+        break;
+      case 'FCP':
+        overrides = {
+          debug_ttfb: attribution.timeToFirstByte,
+          debug_fb2fcp: attribution.firstByteToFCP,
+          debug_load_state: attribution.loadState,
+          debug_target: attribution.loadState || '(not set)',
+        };
         break;
       case 'FID':
-        webVitalInfo = attribution.eventTarget;
+      case 'INP':
+        overrides = {
+          debug_event: attribution.eventType,
+          debug_time: attribution.eventTime,
+          debug_load_state: attribution.loadState,
+          debug_target: attribution.eventTarget || '(not set)',
+        };
         break;
       case 'LCP':
-        webVitalInfo = attribution.element;
+        overrides = {
+          debug_url: attribution.url,
+          debug_ttfb: attribution.timeToFirstByte,
+          debug_rld: attribution.resourceLoadDelay,
+          debug_rlt: attribution.resourceLoadTime,
+          debug_erd: attribution.elementRenderDelay,
+          debug_target: attribution.element || '(not set)',
```

Comment:
Actually this probably won't work

Reply:
Why not?

Reply:
Tested it and it appears to work. E.g. http://localhost:8080/reports/page-weight#bytesFont

Reply:
I was hoping to capture the scroll position at LCP time, but this is the scroll position at the time of reporting.

Reply:
Ah true as the PO is not guaranteed to be emitted immediately. However, in most cases in my experience, it is emitted pretty quickly afterwards so report time is close. So agree we probably shouldn’t add it to web-vitals.js because of that, but could still be an interesting metric to look at for us.

Reply:
Ok, if it can't hurt let's keep it